internal/vlan/create.go

```diff
@@ -45,28 +46,30 @@ func (c *Client) Create() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-
-			req := &packngo.VirtualNetworkCreateRequest{
-				ProjectID: projectID,
-				Metro:     metro,
-				Facility:  facility,
-				VXLAN:     vxlan,
+			virtualNetworkCreateInput := metal.NewVirtualNetworkCreateInput()
+			if metro != "" {
+				virtualNetworkCreateInput.SetMetro(metro)
+			}
+			if facility != "" {
+				virtualNetworkCreateInput.SetFacility(facility)
 			}
+			virtualNetworkCreateInput.SetVxlan(int32(vxlan))
+
 			if description != "" {
-				req.Description = description
+				virtualNetworkCreateInput.Description = &description
 			}
 
-			n, _, err := c.Service.Create(req)
+			n, _, err := c.Service.CreateVirtualNetwork(context.Background(), projectID).VirtualNetworkCreateInput(*virtualNetworkCreateInput).Include(nil).Exclude(nil).Execute()
 			if err != nil {
 				return fmt.Errorf("Could not create ProjectVirtualNetwork: %w", err)
 			}
 
 			data := make([][]string, 1)
 
 			// TODO(displague) metro is not in the response
```

Comment:
This TODO can probably be removed now

Reply:
omitting FacilityCode based on https://github.com/equinix-labs/metal-go/issues/117 and the deprecation of facilities is reasonable.

Additionally, since the introduction of metros, VLAN's are not facility scoped. They are metro scoped. Removing the confusing facility detail makes good sense.